Sources/OpenAPI/.create-api.yml

```diff
@@ -1,46 +1,21 @@
-# Modifier for all generated declarations
-access: public
-# Add @available(*, deprecated) attributed
-isAddingDeprecations: true
-# Available values: ["spaces", "tabs"]
+generate: [entities, paths, enums]
```

Comment:
This defaults to `[entities, paths, enums, package]` so we have to redeclare it to remove `package` from generation. 


This is the equivalent to using `--module` previously